src/js/burials/config/form.js

```diff
@@ -67,6 +79,66 @@ const formConfig = {
           }
         }
       }
+    },
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        veteranInformation: {
+          title: 'Deceased Veteran information',
+          path: 'veteran-information',
+          uiSchema: {
+            'ui:title': 'Deceased Veteran information',
+            veteranFullName: fullNameUI,
+            'view:veteranId': personId.uiSchema(
+              'veteran',
+              'view:veteranId.view:noSSN',
+              'I don’t have the Veteran’s Social Security number'
```

Comment:
Should "Number" in "Social Security Number" be capitalized throughout? You did not introduce this convention here, just wondering if we should follow the labeling instructions @stacyhlinn provided more closely.